controller/array_action/array_mediator_interface.py

```diff
@@ -321,6 +321,22 @@ def get_host_by_host_identifiers(self, initiators):
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def get_host_by_name(self, host_name):
+        """
+        This function will find the host name by iscsi iqn or fc wwns.
```

Comment:
find X by Y means that it will get X using a given Y, so I think this sentence should be the other way around

Reply:
done